test/export/test_export.py

```diff
@@ -6021,7 +6021,7 @@ def forward(self, x):
             """\
 def forward(self, x):
     cos = torch.ops.aten.cos.default(x)
-    auto_functionalized = torch.ops.higher_order.auto_functionalized(torch.ops.testlib.foo.default, x = x, z = cos);  x = cos = None
+    auto_functionalized = torch.ops.higher_order.auto_functionalized(torch.ops.testlib.foo.default, x = x, z = cos, _x_base = x, _z_base = cos, _all_bases = [x, cos]);  x = cos = None
```

Comment:
Why do we need `_x_base`, `_z_base`, and `_all_bases` ?

Reply:
I am changing it to 
[x', y'] = auto_functionalize( foo, x, y, _all_bases=[x,y], _observe_mutation_from=[[arg0],[arg1])
see https://github.com/pytorch/pytorch/pull/134315

Reply:
Ah back to your question, _all_bases removes the repetitions and specify the order of the outputs.